network-api/networkapi/management/commands/load_fake_data.py

```diff
@@ -65,12 +66,16 @@ def handle(self, *args, **options):
         faker.random.seed(seed)
 
         self.stdout.write('Generating LandingPage objects')
-        opportunity = LandingPageFactory.create(title='opportunity', content='This is placeholder')
+        opportunity = LandingPageFactory.create(title='Opportunity', content='This is placeholder')
         [LandingPageFactory.create(parent=opportunity) for i in range(5)]
 
         self.stdout.write('Generating LandingPage objects with Signup CTAs')
         [LandingPageFactory.create(parent=opportunity, signup=SignupFactory.create()) for i in range(5)]
 
+        self.stdout.write('Generating CampaignPage objects')
+        campaigns = LandingPageFactory.create(title='Campaigns', content='This is placeholder')
```

Comment:
The bad grammar in this content has bothered me since I typo'd it originally. can you update that :)

Reply:
How's this?

Reply:
much better!